Mathlib/Analysis/NormedSpace/FunctionSeries.lean

```diff
@@ -5,6 +5,7 @@ Authors: Sébastien Gouëzel
 -/
 import Mathlib.Analysis.Normed.Group.InfiniteSum
 import Mathlib.Topology.Instances.ENNReal
+import Mathlib.Data.Finset.Basic
```

Comment:
Please check if you need this import explicitly; the bot suggests you don't. If not, please remove.